app/uk/gov/hmrc/softdrinksindustrylevy/controllers/SdilController.scala

```diff
@@ -59,9 +59,9 @@ class SdilController @Inject()(desSubmissionService: DesSubmissionService,
   }
 
   def checkPendingSubscription(utr: String): Action[AnyContent] = Action.async { implicit request =>
-    buffer.findById(utr) map {
-      case Some(_) => Ok(Json.obj("status" -> "SUBSCRIPTION_PENDING"))
-      case _ => NotFound(Json.obj("status" -> "SUBSCRIPTION_NOT_FOUND"))
+    buffer.find({"subscription.utr" -> utr}) map {
```

Comment:
You don't need the curly braces here

Reply:
I think this equates better to the representation in Mongo. Arguably more readable too.